packages/angular/src/generators/setup-mf/lib/update-host-app-routes.ts

```diff
@@ -34,12 +34,17 @@ export function updateHostAppRoutes(tree: Tree, options: Schema) {
 `
   );
 
-  const pathToHostRootRoutingFile = joinPathFragments(
+  let pathToHostRootRoutingFile = joinPathFragments(
     sourceRoot,
     'app/app.routes.ts'
   );
 
-  const hostRootRoutingFile = tree.read(pathToHostRootRoutingFile, 'utf-8');
+  let hostRootRoutingFile = tree.read(pathToHostRootRoutingFile, 'utf-8');
+
+  if (!hostRootRoutingFile) {
+    pathToHostRootRoutingFile = (0, devkit_1.joinPathFragments)(sourceRoot, 'app/app-routing.module.ts');
```

Comment:
Please import `joinPathFragments` from `@nrwl/devkit`.

Reply:
It is already imported

Reply:
It might be imported, but you're not using the imported symbol. You're invoking `(0, devkit_1.joinPathFragments)(...)`. It seems like a copy and paste from a compiled output and it should fail the build. Instead, you should have:

```suggestion
    pathToHostRootRoutingFile = joinPathFragments(sourceRoot, 'app/app-routing.module.ts');
```

Reply:
Ah yes, was testing on a compiled code :), my bad, resolved!